secure/secure.sh

```diff
@@ -71,6 +74,12 @@ readonly keystore_password=$(gsutil cat $keystore_password_uri | \
   --ciphertext-file - \
   --plaintext-file - \
   --key $kms_key_uri)
+readonly trust_password=$(gsutil cat $cross_realm_trust_password_uri | \
```

Comment:
Similarly, this needs to be hidden behind an if statement

Reply:
Done.